CODEOFCONDUCT.MD

```diff
@@ -0,0 +1,9 @@
+# Microsoft Open Source Code of Conduct
```

Comment:
should we maybe add this to README.md?

Reply:
@marcocitus I think we should do both. I think we should add a specific, separate file that's CODEOFCONDUCT.md to the repo so that the Code of Conduct appears at the root to external users, and likely also add the language to the README as well. Shall I propose a PR to modify the README text as well?